ansible/group_vars/orange_pi_zero

```diff
@@ -0,0 +1,3 @@
+---
+nginx_worker_processes: 1
```

Comment:
That advice only applies to production scale web services. nginx uses a mostly threadless architecture; worker_processes doesn't adjust the number of simultaneous connections (aka the nginx worker_connections config), just the number of sub-processes that will be listening for new connections. Leaving worker_processes at 1 might actually improve performance in this case since the other cores will be freed up for disk I/O, TCP processing, etc. [Linode agrees](https://www.linode.com/docs/websites/nginx/configure-nginx-for-optimized-performance) Should I put all that in a comment? ;)